addons/l10n_in_edi/models/res_config_settings.py

```diff
@@ -19,4 +19,4 @@ class ResConfigSettings(models.TransientModel):
     def l10n_in_edi_test(self):
         self.env["account.edi.format"]._l10n_in_edi_authenticate(self.company_id)
         if not self.company_id.sudo()._l10n_in_edi_token_is_valid():
-            raise UserError("Username/password are incorrect.")
+            raise UserError("Incorrect username or password, or the GST number on company does not match.")
```

Comment:
```suggestion
            raise UserError(_("Incorrect username or password, or the GST number on company does not match."))
``` but the .pot might need more updating and is not really used anyways